catkin_tools/context.py

```diff
@@ -411,20 +446,20 @@ def summary(self, notes=[]):
                 "--init`.")]
         if not self.source_space_exists():
             summary_warnings += [clr(
-                "Source space `@{yf}{_Context__source_space_abs}@|` does not yet exist.")]
+                "Source space `@{yf}{__source_space_abs}@|` does not yet exist.")]
```

Comment:
Yeah, I'm not really sure what is going on here, but the behaviour is definitely correct in that each instance of `Context` has its own set of these properties, eg:

```
$ ~/env/bin/python
Python 2.7.6 (default, Oct 26 2016, 20:30:19)
[GCC 4.8.4] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from catkin_tools.context import Context
>>> Context.setup_space_keys()
>>> c = Context('.')
>>> c.source_space
'src'
>>> d = Context('.')
>>> d.source_space
'src'
>>> c.source_space = 'foo'
>>> c.source_space
'foo'
>>> c.source_space_abs
'/home/mikepurvis/test_ws/foo'
>>> d.source_space
'src'
```